content/TrueCommand/Administration/Settings/SAMLAD.md

```diff
@@ -0,0 +1,216 @@
+After accessing the server hosting AD, you must:
+
+* Create an AD FS Relying Party Trust.
+
+* Download and modify your TrueCommand system certificate. 
+  Each TrueCommand has a unique certificate you must use for the configuration to work. 
+
+* Configure the TrueCommand URL in Active Directory as a trusted URL
+
+* Configure the identifiers.
+
+* Modify the Relying Party Trust properties to add and edit endpoints.
+
+* Configure the Claim Issuance Policy to add the incoming and outgoing claim types. 
+
+This process is described in detail in the next section.
+
+## Configuring Active Directory as an Identity Server for SAML Service
+
+1. Log into your Active Directory server as an administrator user (*QE* is the administrator user in this example procedure).
+
+2. Create an AD FS Relying Party Trust. Go to **Tools** and select **AD FS Management**. 
+
+   a. Go to **Trust Relationships > Relying Party Trusts** and then delete any entries found. 
+      Each TrueCommand has a unique certificate. To ensure you have the correct TrueCommand certificate, delete existing certificates and then obtain a new one.
+
+   b. Select **Add Relying Party Trust** on the **AD FS** to open the **Add Relying Party Trust Wizard**. Click **Start**.
+      
+      ![AddRelyingPartyTrustWizardWelcome](/images/SAML/AddRelyingPartyTrustWizardWelcome.png "Add Relying Party Trust Wizard Welcome")
+
+   c. Select **Enter data about the relying party manually**, then click **Next**.
+      
+      ![WizardSelectEnterDataAboutRelyingPartyManuallyCropped](/images/SAML/WizardSelectEnterDataAboutRelyingPartyManuallyCropped.png "Wizard Select Enter Data About Relying Party Manually Cropped")
+
+   d. <a name=rpt-name></a>Enter the name for the Relying Party Trust in the **Display Name** field. The example uses *QE SAML*. 
+      QE is the Windows administrator name, and SAML is the service name.
+      
+      ![WizardTypeDisplayNameCropped](/images/SAML/WizardTypeDisplayNameCropped.png "Wizard Type Display Name Cropped")
+   
+   e. Click **Next** to move on to the **Configure Certificate** window.
+
+3. Modify the TrueCommand Certificate (<file>tc.cer</file>). 
+
+   a. Open PowerShell and type the command `invoke-webRequest -uri http://IP:PORT/saml/metadata -outfile tc.cer`. 
+      *IP:PORT* is your TrueCommand system IP address/port number.
+
+      ![OpenPowershellTypeInvokeCommand](/images/SAML/OpenPowershellTypeInvokeCommand.png "Open Powershell Type Invoke Command")
+
+   b. Edit the certificate. 
+
+      1. Open a File Explorer window and locate the <file>tc.cer</file> file in <file>C:/local data/user/QE</file>.
+
+         ![LocateTcCertificateWithFileExplorer](/images/SAML/LocateTcCertificateWithFileExplorer.png "Locate TC Certificate With File Explorer")
+      
+      2. Select the <file>tc.cer</file> file, right-click, then select **Open with Notepad**.
+
+      3. Delete everything before the certificate open tag `<x509Certificate>`. 
```

Comment:
My instructions, in the test case say:
Delete everything except for what is inside X509Certificate, including these tags.

Reply:
Yes, but that URL was just inside the open certificate tag so I had to find a way to describe this and cover that URL if it happens for the user.